plugin/clang.vim

```diff
@@ -152,6 +152,22 @@ func! s:PLog(head, info)
   echom printf("Clang: log: %s >>> %s", string(a:head), string(a:info))
 endf
 "}}}
+"{{{ s:PDumpfile
+"@data list
+"@filename string
+func! s:PDumpfile(data, filename)
+  let l:lv = 5
+  let l:is_windows = has('win32') || has('win64')
+  if l:lv <= g:clang_debug && !l:is_windows
+    let l:fn = a:filename
+    if l:fn == ''
+      let l:fn = 'vim-clang-debug.log'
+    endif
+    let l:fullpath = '/tmp/'.l:fn
+    call writefile(a:data, l:fullpath)
+  endif
+endf
```

Comment:
This function is different from any other Pxxx functions(P here means print). So I don't think is a good idea that we put some debug info into messages and some other into a file, may be confused. I recommend to use `PDebug`